packages/styletron-standard/src/index.ts

```diff
@@ -8,19 +6,10 @@ import type {
 
 export type {FontFaceObject, KeyframesObject};
 
-// Note: $Shape is needed to make polymorphic withStyle refinements work correctly
-// It seems functions satisfy this type without $Shape
-// See: https://github.com/facebook/flow/issues/6784
-//
-//
-//
-//
-//
-//
-export type StyleObject = $Shape<{
-  ...Properties,
-  [string]: StyleObject, // Unrecognized properties are assumed to be media queries or pseudo selectors w/ nested style object. See: https://github.com/styletron/styletron-standard
-}>;
+export interface NestedStyleObject {
+  [x: string]: StyleObject;
+}
+export type StyleObject = NestedStyleObject | Properties;
```

Comment:
This feels inaccurate, but maybe it's my unfamiliarity with TS. Can I write a style object like below? It looks to me that it would not allow combining properties + nested properties

```
{
  backgroundColor: 'blue',
  ':hover': {
    backgroundColor: 'red',
  }
}
```

Reply:
actually, what is currently in a code looks to work the same way as in comment, and should be good for the use case we have: [playground with example](https://www.typescriptlang.org/play/index.html#code/C4TwDgpgBACgTge0nYBLCBnKBeWBDFVPAGwB4BvKAKClqgGMFiE4AuKDYOVAOwHMA3DToAjPPQDWfRAFceAEwDCTFu07d+QgL4A+KlV7AIcAGbjoAOUxH5AZVDEIAeREArCPWBRywgNoAPNS5ePgBddnsQRxd3T219UEgoSOi3Dy9cK04IOwdnNM8oAB9YRGQ0TCEqRh5OKCNOCLyY9JxvYTFJaQQ5JRU2KAAiEWIZCEGAGmFB1gALBAA3Yxn2uihOqVkFZWYBwbgcyeFaGfE0JZWfNdpGXfZ9w6m1rSeoF6otKpq6huAAJiaUXysQyqygAHpwVAAALADAAWgg-kgnkRcEQcA64k2PW2-XYAEY-gBmV4zeZLOCXY7rbHdXo7VRDA7yI5rU6eVAXdhXa6QmFwxHI9JojE0m74qAE160d6yqafBLWAB0txYQn5sIRSJRwFFLCov18AHI5otjMbQsqNvS8bsgA)

Reply:
👍 the playground example lgtm